.github/workflows/e2e-tests.yaml

```diff
@@ -46,6 +46,9 @@ jobs:
       - name: Run tests
         run: pnpm e2e:ci
 
+      - name: Post Cypress – Stop Xvfb server
```

Comment:
Not part of this PR, still okay anyway

Reply:
Yes, it was for testing. It was cherry-picked so it has disappeared now that I updated the branch with main.